projects/1.18/assets/female-gender-forge/wildfire_gender/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+	"category.wildfire_gender.generic": "Wildfire 的性别模组",

```

```suggestion
	"category.wildfire_gender.generic": "Wildfire 的女性模组",
```
额，原文强调了`female`，要不要考虑改一下？